var/spack/repos/builtin/packages/py-pillow-simd/package.py

```diff
@@ -21,3 +21,5 @@ class PyPillowSimd(PyPillowBase):
 
     for ver in ['6.2.2.post1', '7.0.0.post3', '9.0.0.post1']:
         provides('pil@' + ver, when='@' + ver)
+
+    conflicts('target=aarch64:')
```

Comment:
I think this conflict is correct, given that in their GitHub repo they say:

> Currently, Pillow-SIMD can be [compiled](https://github.com/uploadcare/pillow-simd#installation) with SSE4 (default) or AVX2 support.

So apparently only `x86_64` SIMD support, no NEON.